### PR TITLE
Save folders

### PR DESCRIPTION
**Pull request by Maxis010**
- What is the purpose of the pull request?
Converts save files into save folders, vastly reduces save time by only saving NPCs that have changed since the last save

- Give a brief description of what you changed or added.
Changed all save & load methods to support loading both formats but only save to folder format

- Are any new graphical assets required?
No

- Has this change been tested? If so, mention the version number that the test was based on.
Yes, although I would recommend more testing and STRONGLY recommend users back up their saves when this gets added

- So we have a better idea of who you are, what is your Discord Handle?
Maxis010#3060

- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

### CODE REVIEW

File: src/com/lilithsthrone/main/Main.java [hunk 1]
```diff
@@ -796,25 +796,15 @@ public static String getQuickSaveName() {
 		if(!Main.game.isStarted()) {
 			return "QuickSave_intro";
 		}
-		return "QuickSave_"+Main.game.getPlayer().getName(false);
+		return "QuickSave_"+Main.game.getPlayer().getSurname();
```
Comment (AceXP) [inline src/com/lilithsthrone/main/Main.java:799]: Can surname be empty for the player? If so, need an additional check for that.

Comment (Maxis010) [reply src/com/lilithsthrone/main/Main.java:799]: Surname can be an empty string, at which point this would become Quicksave_

**Comment by AceXP** [comment]
This PR is now combined with #1559 Offspring Seed into the single PR #1563.

**Comment by Maxis010** [comment]
Closing PR as I have added further updates to #1563 with AceXP

**Pull request closed without merging**